app/models/condition.rb

```diff
@@ -120,12 +120,12 @@ def self._subst(rec, opts, tag, mode)
       else
         value = ref.nil? ? "" : Tag.list(ref, :ns => tag)
       end
-      value = MiqExpression.quote(value, ohash[:type] || "string")
+      value = MiqExpression.quote(value, ohash[:type]&.to_sym)
```

Comment:
this has been changed to drop the default to `string` (2 places).

But since string and nil should do the same thing in `quote()` and there are other places in the code that call `quote()` without defaulting to string, I decided to drop